File: app/src/main/java/com/example/kitt_kotlin/MainActivity.java
package com.example.kitt_kotlin;


import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;
import android.content.DialogInterface;
import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }

    public void ShowDialog(View view){
        AlertDialog.Builder builder = new AlertDialog.Builder(this);
        builder.setTitle("Title");
        builder.setMessage("Message");
        builder.setPositiveButton("OK", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {

            }
        });
        builder.setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {

            }
        });
        AlertDialog dialog = builder.create();
        dialog.show();
    }
    public void TimeClick(View view){
        Intent i=new Intent(MainActivity.this,Time.class);
        startActivity(i);

    };
    public void DateClick(View view){
        Intent i=new Intent(MainActivity.this,Date.class);
        startActivity(i);

    };
    public void CheckClick(View view){
        Intent i=new Intent(MainActivity.this,Check.class);
        startActivity(i);

    }
    public void ListClick(View view) {
        Intent i = new Intent(MainActivity.this, List.class);
        startActivity(i);
    }
    @Override
    public boolean onCreateOptionsMenu(Menu menu) {

        getMenuInflater().inflate(R.menu.main_menu, menu);
        return true;
    }
    public void FragClick(View view) {
        Intent i = new Intent(MainActivity.this, Fragment.class);
        startActivity(i);
    }
    public void FileClick( View view){
        Intent i= new Intent( MainActivity.this, File_c.class);
        startActivity(i);
    }
    public void BdClick(View view){
        Intent i=new Intent(MainActivity.this, bd_activity.class);
        startActivity(i);
    }
}

File: app/src/main/java/com/example/kitt_kotlin/MainActivity.kt


package com.example.kitt_kotlin

import android.content.Intent
import android.os.Bundle
import android.view.Menu
import androidx.appcompat.app.AlertDialog
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun ShowDialog() {
        val builder = AlertDialog.Builder(this)
        builder.setTitle("Title")
        builder.setMessage("Message")
        builder.setPositiveButton("OK") { dialog, which -> }
        builder.setNegativeButton("Cancel") { dialog, which -> }
        val dialog = builder.create()
        dialog.show()
    }

    fun TimeClick() {
        val i = Intent(this@MainActivity, Time::class.java)
        startActivity(i)
    }

    fun DateClick() {
        val i = Intent(this@MainActivity, Date::class.java)
        startActivity(i)
    }

    fun CheckClick() {
        val i = Intent(this@MainActivity, Check::class.java)
        startActivity(i)
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        menuInflater.inflate(R.menu.main_menu, menu)
        return true
    }

    fun FragClick() {
        val i = Intent(this@MainActivity, Fragment::class.java)
        startActivity(i)
    }

    fun FileClick() {
        val i = Intent(this@MainActivity, File_c::class.java)
        startActivity(i)
    }

    fun BdClick() {
        val i = Intent(this@MainActivity, bd_activity::class.java)
        startActivity(i)
    }

    fun jsoClick() {
        val i = Intent(this@MainActivity, json::class.java)
        startActivity(i)
    }
}